crates/turbopack-cli/src/main.rs

```diff
@@ -1,28 +1,77 @@
 #![feature(future_join)]
 #![feature(min_specialization)]
 
-use anyhow::Result;
+use std::{borrow::Cow, path::Path};
+
+use anyhow::{Context, Result};
 use clap::Parser;
+use tracing_appender::non_blocking::DEFAULT_BUFFERED_LINES_LIMIT;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt, EnvFilter, Registry};
 use turbopack_cli::{arguments::Arguments, register};
+use turbopack_cli_utils::{exit::exit_guard, raw_trace::RawTraceLayer};
 
 #[global_allocator]
 static ALLOC: turbo_tasks_malloc::TurboMalloc = turbo_tasks_malloc::TurboMalloc;
 
 fn main() {
     use turbo_tasks_malloc::TurboMalloc;
 
-    let subscriber = Registry::default();
+    let args = Arguments::parse();
+
+    let trace = std::env::var("TURBOPACK_TRACING").ok();
+
+    let _guard = if let Some(mut trace) = trace {
+        // Trace presets
+        match trace.as_str() {
+            "overview" => {
+                trace = "turbo_tasks_fs=info,turbopack_dev_server=info".to_string();
+            }
+            "turbopack" => {
+                trace = "turbo_tasks_fs=info,turbopack=trace,turbopack_core=trace,\
+                         turbopack_ecmascript=trace,turbopack_css=trace,turbopack_dev=trace,\
+                         turbopack_image=trace,turbopack_dev_server=trace,turbopack_json=trace,\
+                         turbopack_mdx=trace,turbopack_node=trace,turbopack_static=trace,\
+                         turbopack_cli_utils=trace,turbopack_cli=trace,turbopack_ecmascript=trace"
+                    .to_string();
+            }
+            "turbo-tasks" => {
+                trace = "turbopack=trace,turbopack_core=trace,turbopack_ecmascript=trace,\
+                         turbopack_css=trace,turbopack_dev=trace,turbopack_image=trace,\
+                         turbopack_dev_server=trace,turbopack_json=trace,turbopack_mdx=trace,\
+                         turbopack_node=trace,turbopack_static=trace,turbopack_cli_utils=trace,\
+                         turbopack_cli=trace,turbopack_ecmascript=trace,turbo_tasks=trace,\
+                         turbo_tasks_viz=trace,turbo_tasks_memory=trace,turbo_tasks_fs=trace"
+                    .to_string();
+            }
```

Comment:
I feel like constructing these strings from a list would be a lot more readable, and it's only done once anyway?